### PR TITLE
Add DOIT-BI organization, source and category

### PR DESCRIPTION
**Pull request by doit-bi**

### CODE REVIEW

**Comment by doit-bi** [comment]
@googlebot I signed it!

**Comment by doit-bi** [comment]
@googlebot I fixed it

**Comment by doit-bi-do** [comment]
@googlebot I consent.

**Review by doit-bi-do — DISMISSED** [review]

**Review by doit-bi-do — COMMENTED** [review]
ok

**Review by doit-bi-do — COMMENTED** [review]
All okey

**Comment by doit-bi** [comment]
@pfrisella Could you review and merge please

File: organizations/doit-bi.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (pfrisella) [inline organizations/doit-bi.json:1]: make the filename underscores to match the ID. doit_bi.json

File: sources/doit-bi.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (pfrisella) [inline sources/doit-bi.json:1]: make the filename underscores to match the ID. doit_bi.json

**Review by pfrisella — CHANGES_REQUESTED** [review]

**Comment by doit-bi** [comment]
@pfrisella I renamed files. Can you check?

**Review by pfrisella — APPROVED** [review]
LGTM. Thanks for adding!

**Pull request merged**